data/json/items/containers/containers.json

```diff
@@ -2056,6 +2056,33 @@
     "flags": [ "FRAGILE_MELEE" ],
     "melee_damage": { "bash": 2 }
   },
+  {
+    "type": "GENERIC",
+    "category": "container",
+    "id": "clay_cup",
+    "name": { "str": "clay cup" },
+    "symbol": "u",
+    "description": "A simple clay cup used for holding liquid.",
+    "weight": "318 g",
+    "//1": "Weight calculated proportionally from the coffee mug as I couldn't find detailed weight information for the product.",
+    "volume": "475 ml",
+    "longest_side": "12 cm",
+    "price": "5 USD",
+    "to_hit": { "grip": "solid", "length": "hand", "surface": "any", "balance": "clumsy" },
```

Comment:
```suggestion

```

Unnecessary when the item cannot be used as a weapon. Unless you wanna give it like a single bash point of damage and ``FRAGILE_MELEE``